locales/en-US/messages.properties

```diff
@@ -60,6 +60,7 @@ navTextSize=Text size
 navColorTheme=Color theme
 navWrapText=Wrap long lines
 navAllowJS=Allow JavaScript
+navAutoComplete=Autcomplete code
```

Comment:
See my feedback in the Brackets bug about "Autocomplete" vs. "AutoComplete" vs. "Auto Complete" vs. "Auto-Complete" etc.  Let's standardize, and I'd suggest "Autocomplete."